requirements-dev.txt

```diff
@@ -8,7 +8,7 @@ flake8==2.4.0
 pytest-django==2.8.0
 pytest-flakes==0.2
 pytest-pythonpath==0.6
-git+git://github.com/mverteuil/pytest-ipdb.git
+# git+git://github.com/mverteuil/pytest-ipdb.git
```

Comment:
Please do not change things that do not relate to the PR you have open. You can always create a new issue/PR for changing such things. You can undo this change.